tools/ci-ts/tests/flux-monitor.test.ts

```diff
@@ -105,6 +105,7 @@ describe('flux monitor eth client integration', () => {
     fluxMonitorJob.initiators[0].params.address = fluxAggregator.address
     job = clClient.createJob(JSON.stringify(fluxMonitorJob))
     assert.equal(clClient.getJobs().length, initialJobCount + 1)
+    await wait(10000)
```

Comment:
we shouldn't have to artificially wait here because the `assertJobRun` func already asserts over a period of 20 seconds